src/points/PointsDiscoverCard.tsx

```diff
@@ -36,63 +36,68 @@ export default function PointsDiscoverCard() {
   }
 
   return (
-    <View style={styles.container}>
-      <Touchable
-        style={styles.touchable}
-        borderRadius={Spacing.Smallest8}
-        onPress={handlePress}
-        testID="PointsDiscoverCard"
-      >
-        <>
-          <View style={styles.header}>
-            <Text style={styles.title}>{t('points.discoverCard.title')}</Text>
-            <View style={styles.pill}>
-              <Text style={styles.balance}>{pointsBalance}</Text>
-              <LogoHeart size={Spacing.Regular16} />
-            </View>
-          </View>
+    <Touchable
+      shouldRenderRippleAbove
```

Comment:
Why's the `shouldRenderRippleAbove` here now?

Reply:
it's because it ripples beneath the image and the pill otherwise (see below). 
I would be glad for any suggestions on how to solve that in another way 😅

| `shouldRenderRippleAbove=true` | `=false`  |
|--------|--------|
| ![](https://github.com/valora-inc/wallet/assets/2737872/8ff407ce-cfb7-4cfb-8216-6965c253cf84) | ![](https://github.com/valora-inc/wallet/assets/2737872/ff08b087-39e7-4522-a8d2-92a5bff1fcd0) |

Reply:
ahh got it, ty!